_docs-sources/library/usage/contributing.md

```diff
@@ -2,29 +2,36 @@
 sidebar_label: "Contributing"
 ---
 
-# Contributing to the Gruntwork Infrastructure as Code Library
+# Contributing to Gruntwork Library
 
-Contributions to the Gruntwork Infrastructure as Code Library are very welcome and appreciated! If you find a bug or want to add a new
-feature or even contribute an entirely new module, we are very happy to accept
-[pull requests](https://help.github.com/articles/about-pull-requests/), provide feedback, and run your changes through
-our automated test suite.
+Contributions to Gruntwork Library are welcome and appreciated! In fact, Gruntwork receives over 1,000 customer [pull requests](https://help.github.com/articles/about-pull-requests/) per year.
 
-This section outlines the process for contributing.
+You can contribute to the library in several ways:
 
-## File a GitHub issue
+- Update an existing "building block" module or service module
+- Contribute a new module
+- Document a bug
 
-Before starting any work, we recommend filing a GitHub issue in the appropriate repo. This is your chance to ask
+## Start by filing a GitHub issue
+
+If you're thinking of adding a new feature, before starting any work, we recommend filing a GitHub issue in the appropriate repo. This is your chance to ask
 questions and get feedback from the maintainers and the community before you sink a lot of time into writing (possibly
 the wrong) code. If there is anything you’re unsure about, just ask!
 
-## Update the documentation
+If you're submitting a simple change such as a new variable or new output value, it may be worth opening a pull request directly, as described below. The key point is that we want to minimize the risk that you invest time in something we ultimately can't use.
```

Comment:
```suggestion
If you're submitting a simple change such as a new variable or new output value, it may be worth opening a pull request directly, as described below.
```

If you want to add the "minimize risk that you invest time..." line, it should be in the first paragraph in this section, before "If there is anything you're unsure about...".